### PR TITLE
Create div-calculator.rb

### PR DESCRIPTION
**Pull request by Naltrexone**
# Calculator
Congratulations! You're submitting your assignment.

## Comprehension Questions

|  Question 	|  Answer 	|
|---	|---	|
|  Describe how you stored user input in your program?	|   By using gets.chomp & storing in a variable	|
|  How did you determine what operation to perform?	|   By using the user input and accounting for user input being in words or as a numerical operator	|
|  Do you feel like you used consistent indentation throughout your code? 	|   Yes, I tried to.	|
|  If you had more time, what would you have added to or changed about the program? 	| Would have made sure that it accounted for every corner case depending on user input.   	|

### CODE REVIEW

File: div-calculator.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# calculates answer after taking user input for operation to be 
+  
+# checks to make sure that symbol is valid
+def symbol_entry(symbol)
+  if symbol != "add" && symbol != "+" && symbol != "subtract" && symbol != "-" && symbol != "multiply" && symbol != "*" && symbol != "divide" && symbol != "/" && symbol != "^" && symbol != "%" && symbol != "exponent" && symbol != "module"
+    puts "Invalid symbol input!"
+  end 
+end
```
Comment (dHelmgren) [inline div-calculator.rb:8]: Test comment!

**Comment by Hamled** [comment]
Closing this PR as it should be submitted against the [Ada-C10/calculator](https://github.com/Ada-C10/calculator) fork instead.

File: div-calculator.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ 
+# does the math according to numerical operator  
+def symbol_check(symbol, value1, value2) 
+print "Answer: " 
+  case symbol
+  when "add", "+"
+    print "#{value1} + #{value2} = " 
+    ans = value1 + value2
```
Comment (dHelmgren) [inline div-calculator.rb:16]: You're doing both your calculations and your print statements in the same method! The further you get into the course, the more I'd encourage you to make sure your methods are doing just one "operation" (ie, just doing the math, just doing output, just sorting an array).

File: div-calculator.rb [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    print"#{value1} ** #{value2} = " 
+    ans = value1 ** value2 
+  when "%" , "module"
+    print"#{value1} % #{value2} = " 
+    ans = value1 % value2 
+  else 
+    puts "Unable to calculate!"
+  end
+    
+  if ans % 1 > 0.0
```
Comment (dHelmgren) [inline div-calculator.rb:41]: You get an error after the user gives you bad input, but only after your program prints "Unable to calculate!". See if after Thurday's lessons you can fix the error. If you have questions as to how you might go about this, feel free to talk to me about it.

File: div-calculator.rb [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+# prints the options for user to select function
+puts "Welcome to the calculator program! Which operator would you like to use? "
+puts "1. add(+)"
+puts "2. subtract(-)"
+puts "3. multiply(*)"
+puts "4. divide(/)" 
+puts "5. exponent(^)"
+puts "6. module(%)"
+print "Please use one operator name or symbol: "
+symbol = gets.chomp
```
Comment (dHelmgren) [inline div-calculator.rb:58]: From a user's perspective, it's a bit clunky to collect the symbol and then the numbers, and **_only then_** check whether the data is acceptable. It works, but keep in mind the user's experience as you work on future projects.

**Comment by Naltrexone** [comment]
Thanks for the feedback Devin! Now that I'm more familiar with scope, I'm
hoping that I can refine my code by splitting it into different functions.
Will try refactoring it.

**Pull request closed without merging**